pages/trip/[id].js

```diff
@@ -43,6 +46,8 @@ export default function TripDetails() {
       <br />
       {title}
       <br />
+      <Counter startDate={startDate} endDate={endDate} />
+      <br />
```

Comment:
Again no `<br>`s

Reply:
Again, we didn't had the styling there, it will be merged with another PR ✌️